src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.cs

```diff
@@ -63,9 +63,9 @@ public static bool IsDrawingSupported
         public static bool SupportsSsl3 => GetSsl3Support();
 
 #if NETCOREAPP
-        public static bool IsReflectionEmitSupported = RuntimeFeature.IsDynamicCodeSupported;
+        public static bool IsReflectionEmitSupported { get; } = RuntimeFeature.IsDynamicCodeSupported;
```

Comment:
```suggestion
        public static bool IsReflectionEmitSupported => RuntimeFeature.IsDynamicCodeSupported;
```

to match the style of other properties in this file

Reply:
Good point - fixed. In case it wasn't clear too the reason this had to changed to a property was so that it could be used by `ConditionalFact`

Reply:
Regarding `{get;} =>` vs. `=>`, if `IsDynamicCodeSupported` does not change during the lifetime of test execution, should it use the former syntax?